src/components/shared/MarkerTooltipContents.js

```diff
@@ -59,7 +59,7 @@ function _markerDetailNullable<T: NotVoidOrNull>(
   value: T | void | null,
   fn: T => string = String
 ): React.Node {
-  if (value === undefined || value === null) {
+  if (value === undefined || value === null || value.length === 0) {
```

Comment:
`value` is of generic type `T` and not necessarily a string, so for non-string values we would check the `length` property on something that's not a string... which isn't the end of the world, because accessing non-existent properties on a random object won't cause an error, but it's not the prettiest thing either.

Could you check `fn(value).length` instead? That would check the string that gets displayed for the value. And maybe add a comment above the function definition that says "Returns null if the value is undefined, null, or the displayed string would be empty".